assemblies/assembly_install-on-tomcat/main.adoc

```diff
@@ -24,9 +24,9 @@ include::{enterprise-dir}/installation/install-download-proc.adoc[leveloffset=+1
 include::{enterprise-dir}/installation/assembly_installing-using-installer.adoc[leveloffset=+1]
 
 == {KIE_SERVER} ZIP file installation and configuration
-You can install {KIE_SERVER} using the `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip` file
+You can install {KIE_SERVER} using the `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip` file available from the *{PRODUCT} {PRODUCT_VERSION_LONG} Add Ons* (`{PRODUCT_FILE}-add-ons.zip`) file on the Customer Portal
 ifdef::PAM[]
- and then configure the Java Database Connectivity (JDBC) web server data sources on {TOMCAT}
+and then configure the Java Database Connectivity (JDBC) web server data sources on {TOMCAT}
```

Comment:
Missing full stop after {TOMCAT}.

Reply:
Actually, @csherrar the full stop isn't missing. It just looks that way because of the use of conditions. Always check the rendered doc.